usaon_vta_survey/models/tables.py

```diff
@@ -64,6 +64,26 @@ def __io__(cls) -> IORelationship:
         return io
 
 
+class ResponseObjectFieldMixin:
+    """Provide shared fields between all relationshio objects to reduce repition."""
+
+    short_name = Column(String(256), nullable=False)
+    full_name = Column(String(256), nullable=False)
+    organization = Column(String(256), nullable=False)
+    # should this be funding agency?
+    funder = Column(String(256), nullable=False)
+    funding_country = Column(String(256), nullable=False)
+    # clairify this
+    website_url = Column(String(256), nullable=True)
+    description = Column(String(512), nullable=True)
+    contact_name = Column(String(256), nullable=False)
+    contact_title = Column(String(256), nullable=True)
+    contact_email = Column(String(256), nullable=False)
+    # how do i make it a minimum of 3 tags?
+    tags = Column(String, nullable=False)
```

Comment:
@rmarow I'm not sure I understand this. I looked in the glossary and didn't see anything about a minimum number of tags. I think this should be an [Array column](https://www.postgresql.org/docs/current/arrays.html) ([sqlalchemy](https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.ARRAY))!

Reply:
If this is hard to code, we can just add explanatory text that says "recommend three or more tags." Sandy had suggested this last week, and I added a note in the front-end document, which @rmarow noted. I think the explanatory text is a good way to go. If we do some testing and folks aren't responding to that request, we can adjust.

Reply:
I agree, let's go with explanatory text and treat the actual validation as icing for later :)